### PR TITLE
0.7.3 changelog

### PR DESCRIPTION
**Pull request by zhujun98**

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,57 @@
...
+    - In the terminal, "--topic" becomes a positional argument. #41
+
+- **Improvement**
+
+    - Reimplement Color classes. mkPen and mkBrush from pyqtgraph are not needed
+      anymore. #53
+    - Allow select pipeline policy (wait or drop) via commandline. The default is wait
+      since the data arrival speed is slower than the processing speed during online
+      analysis. #45
+    - Replace Python's build in queue.Queue to speed up data transfer. #45
```
Comment (ebadkamil) [inline docs/changelog.rst]: built in

File: docs/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,57 @@
...
+    - Implement Fast assembling for LPD and DSSC in C++. #40
+    - Resign the config code. Now each instrument will has its own config file,
+      e.g. scs.config.yaml, fxe.config.yaml. All the instrument sources will be
+      set up in the config file. #38
+    - Implement streaming raw (AGIPD, LPD) data from files and also 'confirmed'
+      streaming raw (AGIPD, LPD) data online. #38
+
+- **New Feature**
+
+    - Allow specify bin range of histogram. #56
```
Comment (ebadkamil) [inline docs/changelog.rst]: Specify bin range ...or Allow specific bin range...

File: docs/changelog.rst (hunk 1 above)
Comment (zhujun98) [reply docs/changelog.rst]: Changed! Thanks!

File: docs/changelog.rst (hunk 2 above)
Comment (zhujun98) [reply docs/changelog.rst]: Changed! Thanks!

**Comment by zhujun98** [comment]
Thanks for the review!

**Pull request merged**